README.md

```diff
@@ -28,7 +28,7 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for info on contributing to Cucumber.
 * Ruby 2.5
 * Ruby 2.4
 * Ruby 2.3
-* JRuby 9.1
+* JRuby 9.1 (with [some limitations](https://github.com/cucumber/cucumber-ruby/blob/master/docs/jruby-limitations.md))
```

Comment:
Perhaps "JRuby 9.1, 9.2"?

```suggestion
* JRuby 9.1, JRuby 9.2 (with [some limitations](https://github.com/cucumber/cucumber-ruby/blob/master/docs/jruby-limitations.md))
```

Reply:
It may be worth checking it still build with JRuby 9.1, but that's just a task to add to CircleCI, should not be much of a trouble.
I'll add that :)